packages/docusaurus/bin/beforeCli.js

```diff
@@ -20,105 +20,114 @@ const {
   engines: {node: requiredVersion},
 } = require('../package.json');
 
-// Notify user if @docusaurus packages is outdated
```

Comment:
😅 I have no idea what's going on here but it doesn't look like just TS typing refactor, hope this doesn't break anything

Reply:
Oh, I just wrapped all the side-effects in a function and called it. This is necessary to remove the `@ts-ignore`s and get the correct types.

Reply:
If you review with "hide whitespace" on, you will get a much more meaningful diff.